src/index.js

```diff
@@ -4,7 +4,18 @@ import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+//Redux
+import { Provider } from 'react-redux';
+import store from './ducks/store';
+
+//Routing
+import { HashRouter } from 'react-router-dom'; 
```

Comment:
Please update HashRouter to BrowserRouter.

Reply:
And maybe form it as 
`import { BrowserRouter as Router }`
and update the wrapping of <App/> accordingly